test/index.test.mjs

```diff
@@ -153,6 +154,25 @@ test('ProcessPromise', async () => {
   assert.equal(err.message, 'The pipe() method does not take strings. Forgot $?')
 })
 
+test('ProcessPromise: inherits native Promise', async () => {
+  const p1 = $`echo 1`
+  const p2 = p1.then(v => v)
+  const p3 = p2.then(v => v)
+  const p4 = p3.catch(v => v)
+  const p5 = p1.finally(v => v)
+
+  assert.ok(p1 instanceof Promise)
+  assert.ok(p1 instanceof ProcessPromise)
+  assert.ok(p2 instanceof ProcessPromise)
+  assert.ok(p3 instanceof ProcessPromise)
+  assert.ok(p4 instanceof ProcessPromise)
+  assert.ok(p5 instanceof ProcessPromise)
+  assert.ok(p1 !== p2)
```

Comment:
I'm not getting this test. What do we test here?

Reply:
It's aimed to check that new promise instances from `thenables` methods are instances of `ProcessPromise` too.